openage/convert/changelog.py

```diff
@@ -35,6 +35,7 @@
     {"graphics"},
     {"interface"},
     {"interface"},
+    {"gamedata"},
```

Comment:
we might wanna purge the old data, but we have no infrastructure for that yet. i.e. removing the thousands of files that will otherwise lay around forever.